### ISSUE TITLE
Write note on selection of aliases

### ISSUE THREAD

**Issue by tlongers** [security-force-monitor/sfm-research-handbook#39]
`Gerardo Rubén Serrano Herrera`
`Also known as: Rubén Serrano Herrera | Serrano Herrera`

Source: http://web.archive.org/web/20170711152120/http://www.zocalo.com.mx/opinion/opi-interna/salvador-rodriguez-orozco-a-la-fiscalia-de-la-sexta-zona-militar

Where a source refers to a person by their surname, we should not include this as an `alias`. The source's use of a surname is an stylistic decision by the publication's editors.

**Comment by tonysecurityforcemonitor** [comment]
Agreed - however the difficultly is determining what is/is not a surname - perhaps for efficiency sake we include everything under the rubric that Alias=every other way to refer to this unit/person
